src/app/features/genes/components/gene-hero/gene-hero.component.ts

```diff
@@ -70,4 +71,15 @@ export class GeneHeroComponent {
     }
     return '';
   }
+
+  getBiodomains(): string {
+    if (!this.gene || !this.gene.bio_domains)
+      return '';
+    console.log(this.gene.bio_domains.gene_biodomains);
+    const biodomains = this.gene.bio_domains.gene_biodomains
+      .filter(b => b.pct_linking_terms > 0)
```

Comment:
Is this a standard filter to apply to the `gene_biodomains`? Just wondering since I didn't see the filter explicitly called out in the ticket

Reply:
Good question! There are 19 biodomains as I recall.  Since we always show all of them regardless in the charts, the non-zero ones should be shown in the gene hero.  I'll ask Jess to add that to the description.